js/translator.js

```diff
@@ -65,13 +65,15 @@ function translate_story(nav) {
   idx_store.innerHTML = idx;
   serial_store.innerHTML = n;
   document.getElementById("number_of_sections").innerHTML = sections.length;
-
   get_storage(idx);
   tr_title.focus();
 
   document.getElementById("rev_btn").innerHTML = '<a href="#modal-review" class="call-modal" onclick="review_translation()">Review submission</a>';
   document.getElementById("review_sub").style.display = '';
 
+  if (typeof window.location.hash !== 'undefined') {
+    window.location.hash = 'story-' + idx;
```

Comment:
I think this and lines `142` and `143` are the only other places that would need to be adjusted. :smile: